prdoc/pr_1454.prdoc

```diff
@@ -0,0 +1,10 @@
+title: Support XCM as part of Cosmos CosmWasm contract messages
```

Comment:
Its not really constrained to Cosmos, or? Just anything compatible with JSON schema should work?

Reply:
and Polkadot Parachain too https://docs.composable.finance/develop/cosmwasm-cli

Reply:
BUT 
that is true - `anything compatible with JSON schema should work` , like OpenAPI/REST/Swagger things sure. 

So yeah - people will be able to send XCM via REST:)

Reply:
I tell Cosmos because want to send XCM message over IBC connect as option or CW contract on Parachain to handle send/receive XCM.

Reply:
@ggwpez 

> anything compatible with JSON schema should work

thank you. yes. that little step XCM to fullfill its promise - multichain multicontract being

Reply:
seems fixed build.